039-AKSEnterpriseGrade/Student/01-containers.md

```diff
@@ -10,7 +10,7 @@ This challenge will cover the basics of containers and container runtimes, and g
 
 Create an Azure Container Registry. Build the API and Web images in this repository and store them in your new ACR.
 
-The application we will use in this hack has three components, as the following picture describes: a [web](./Resources/web) tier offers an HTML portal that shows the information produced by the [api](./Resources/api)], that in its turn access a **database** with a simple query that shows the database version. You will find the source code for each application in the files supplied for this hack:
+The application we will use in this hack has three components, as the following picture describes: a [web](./Resources/web) tier offers an HTML portal that shows the information produced by the [api](./Resources/api), that in its turn access a **database** with a simple query that shows the database version. You will find the source code for each application in the files supplied for this hack:
```

Comment:
@erjosito in this paragraph, you are linking directly to the WTH repo.  

As per the WTH Author's guide, the student guide should not link directly to the repo.  There should be a new line above this paragraph that directs students to download the Resources.zip file provided by their coach.

For example:
"Your coach will provide you with a Resources.zip file that contains the application code and resources you will use to complete the challenges of this hack."

Any subsequent reference to the contents of the /Student/Resources folder can be referred to as a relative path within the Resources.zip file.  Then, in the line above, you should refer to the "api" and "web" folders in the Resources.zip file.

Reply:
+1, Agree with PL's comment.

Reply:
Ha, I had forgotten about this PR :-D! I thought we agreed that if it is markdown, it is fair game? (I could link to `./Resources/web/README.md`. If there is no zip file to distribute, the coach experience is much simpler, so I would really try to avoid to use the zip file. Maybe a solution lies in the comment below, where we put the authoritative site outside of the WTH repo?

Reply:
@erjosito Hmmm...  If you're linking to another MD file, that's okay.  Remember, students will be accessing the hack via our WTH website at: https://aka.ms/wth   This is the Github pages site rendered from the MD files.  They should never see the repo (at least until after the hack is over.)

If you have a readme.md in folder, linking to the folder (like you have on this line) will render that readme.md as a webpage.  That is technically OKAY... since those readme files provide instructions on how to work with the sample app.

However, it does not negate the need for the Resources.zip file to provide the rest of the contents of the /Student/Resources folder to the student.  Remember, they won't see the files on the rendered webpage.

Unfortunately, the original solution we had to provide a link that dynamically generates the Resources.zip file with "DownGit" stopped working (Chrome marked it as malware.)   It is the lead coach's job to package up the Resources.zip file before the hack and provide it to the students. That is documented in the How to Host guide. 

For consistency, we have been advising all authors to refer to relative paths in the "Resources.zip file that your coach has provided".  We've been doing this for the past year or so.  I'm planning to update the author's guide to indicate this more clearly soon.